core/controllers/moderator_test.py

```diff
@@ -25,24 +25,9 @@
 
 
 class ModeratorPageTests(test_utils.GenericTestBase):
-
-    def test_moderator_page(self) -> None:
-        """Tests access to the Moderator page."""
-        # Try accessing the moderator page without logging in.
-        self.get_html_response('/moderator', expected_status_int=302)
-
-        # Try accessing the moderator page without being a moderator.
-        self.signup(self.VIEWER_EMAIL, self.VIEWER_USERNAME)
-        self.login(self.VIEWER_EMAIL)
-        self.get_html_response('/moderator', expected_status_int=401)
-        self.logout()
-
-        # Try accessing the moderator page after logging in as a moderator.
-        self.signup(self.MODERATOR_EMAIL, self.MODERATOR_USERNAME)
-        self.set_moderators([self.MODERATOR_USERNAME])
-        self.login(self.MODERATOR_EMAIL)
-        self.get_html_response('/moderator')
-        self.logout()
+    def test_moderator_get(self) -> None:
+        """Test `/moderator` returns a 200 response."""
+        self.get_html_response('/admin', expected_status_int=200)
```

Comment:
```suggestion
        self.get_html_response('/moderator', expected_status_int=200)
```

Reply:
I think we can remove this test class completely!

Reply:
Done